Mathlib/RingTheory/LocalProperties/IntegrallyClosed.lean

```diff
@@ -16,52 +16,63 @@ In this file, we prove that `IsIntegrallyClosed` is a local property.
 
 * `IsIntegrallyClosed.of_localization_maximal` : An integral domain `R` is integral closed
   if `Rₘ` is integral closed for any maximal ideal `m` of `R`.
-
-## TODO
-
-Prove that `IsIntegrallyClosed` is preserved by localization
```

Comment:
Is this todo really done? Preserved by localization probably means `R` integrally closed => `R_p` integrally closed but I don't see that in this PR.

Reply:
[isIntegrallyClosed_of_isLocalization](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/IntegralClosure/IntegrallyClosed.html#isIntegrallyClosed_of_isLocalization) is already there
(added 7 months ago in https://github.com/leanprover-community/mathlib4/commit/134f2f109f0ac326add237c68eee4d87539f0926)